### PR TITLE
MCLOUD-5982: Added PHP container script (Remove any PHP requirement for Config build)

### PR DESCRIPTION
**Pull request by G-Arvind**
### Description

Remove any PHP requirement that is present for building docker configuration(Added PHP container script)

#### Command changes

- **Removed PHP requirement for Config build**-Use the `./bin/magento-docker php <version> <command>` to run a command inside PHP container . For instance `./bin/magento-docker php 7.3 vendor/bin/ece-docker build:compose` will build the configuration, this happens with only Docker and Bash installed, and does not require PHP on the host system.

### Fixed Issues (if relevant)

https://jira.corp.magento.com/browse/MCLOUD-5982:
1. magento/magento-cloud-docker#148: Remove PHP requirement for Docker configuration build

### Manual testing scenarios

1. Run `./bin/magento-docker php <version> <command>` 

Example:  Run `./bin/magento-docker php 7.3 vendor/bin/ece-docker build:compose` to build configuration 

### Contribution checklist
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages

### CODE REVIEW

File: dist/bin/magento-docker [hunk 1]
```diff
@@ -19,15 +19,30 @@ USAGE="Magento Cloud Docker
   flush-redis       clears redis cache
   flush-varnish     clears varnish cache
   ece-db            access to Database
+  php 7.1           run a command in a PHP 7.1 container
```
Comment (shiftedreality) [inline dist/bin/magento-docker:22]: Can't we have 1 command with `--version` as an option?

Comment (bbatsche) [reply dist/bin/magento-docker:22]: I keep going back and forth on this one myself as well. I do like how clean and clear this (`php <version>`) syntax is, but it goes against typical behavior for command line arguments. In practical terms I don't know if that would be a problem, but it gives me pause.

Comment (G-Arvind) [reply dist/bin/magento-docker:22]: @shiftedreality @bbatsche, I have refactored the code now, removed the switch case inside php option. Can you please take a look at the updated code.

Thanks.

**Review by shiftedreality — APPROVED** [review]

**Review by bbatsche — APPROVED** [review]

**Comment by andriyShevtsov** [comment]
QA approved

**Comment by shiftedreality** [comment]
@G-Arvind could you please add some release notes to description?

**Comment by G-Arvind** [comment]
> @G-Arvind could you please add some release notes to description?

Hi @shiftedreality, Added release notes in the description of this PR. Thanks.

**Pull request merged**